docs/markdown/Creating-Linux-binaries.md

```diff
@@ -9,8 +13,8 @@ First you need to do a fresh operating system install. You can use spare hardwar
 Once you have installed the system, you need to install build dependencies for GCC. In Debian based distros this can be done with the following commands:
 
 ```console
-$ apt get build dep g++
-$ apt get install pkg config libgmp dev libmpfr dev libmpc dev
+$ apt-get build-dep g++
```

Comment:
yeah, good point. I missed the memo that apt now works as-is (on Fedora here). what about build-dep though, does that work without the dash?